diff --git a/android_vendor_FDroid_PrebuiltApps/Android.mk b/android_vendor_FDroid_PrebuiltApps/Android.mk
index ee70d71..a421b49 100644
--- a/android_vendor_FDroid_PrebuiltApps/Android.mk
+++ b/android_vendor_FDroid_PrebuiltApps/Android.mk
@@ -59,66 +59,6 @@ include $(BUILD_PREBUILT)
 ################################################################
 include $(CLEAR_VARS)

-LOCAL_MODULE := F-DroidOfficial
-LOCAL_MODULE_CLASS := APPS
-LOCAL_PRODUCT_MODULE := true
-LOCAL_CERTIFICATE := PRESIGNED
-LOCAL_DEX_PREOPT := false
-
-LOCAL_SRC_FILES := packages/$(LOCAL_MODULE).apk
-
-include $(BUILD_PREBUILT)
-################################################################
-include $(CLEAR_VARS)
-LOCAL_MODULE := permissions_org.fdroid.fdroid.privileged.xml
-LOCAL_MODULE_CLASS := ETC
-LOCAL_PRODUCT_MODULE := true
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_PATH := $(TARGET_OUT_ETC)/permissions
-LOCAL_SRC_FILES := $(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
-
-LOCAL_MODULE := F-DroidPrivilegedExtensionOfficial
-LOCAL_MODULE_CLASS := APPS
-LOCAL_PRODUCT_MODULE := true
-LOCAL_CERTIFICATE := PRESIGNED
-LOCAL_DEX_PREOPT := false
-
-LOCAL_PRIVILEGED_MODULE := true
-LOCAL_REQUIRED_MODULES := permissions_org.fdroid.fdroid.privileged.xml
-
-LOCAL_SRC_FILES := packages/$(LOCAL_MODULE).apk
-
-include $(BUILD_PREBUILT)
-################################################################
-include $(CLEAR_VARS)
-
-LOCAL_MODULE := IchnaeaNlpBackend
-LOCAL_MODULE_CLASS := APPS
-LOCAL_PRODUCT_MODULE := true
-LOCAL_CERTIFICATE := PRESIGNED
-LOCAL_DEX_PREOPT := false
-
-LOCAL_SRC_FILES := packages/$(LOCAL_MODULE).apk
-
-include $(BUILD_PREBUILT)
-################################################################
-include $(CLEAR_VARS)
-
-LOCAL_MODULE := NominatimNlpBackend
-LOCAL_MODULE_CLASS := APPS
-LOCAL_PRODUCT_MODULE := true
-LOCAL_CERTIFICATE := PRESIGNED
-LOCAL_DEX_PREOPT := false
-
-LOCAL_SRC_FILES := packages/$(LOCAL_MODULE).apk
-
-include $(BUILD_PREBUILT)
-################################################################
-include $(CLEAR_VARS)
-
 LOCAL_MODULE := OpenCamera
 LOCAL_MODULE_CLASS := APPS
 LOCAL_PRODUCT_MODULE := true
